// [[Reason: Implicit undefined not fixed by code mod yet]] ////

//// [tests/cases/compiler/isolatedDeclarationsAddUndefined.ts] ////

===================================================================
--- TSC	declarations
+++ DTE	declarations
@@ -10,9 +10,9 @@
 }
 export {};
 //# sourceMappingURL=file1.d.ts.map
 //// [file2.d.ts]
-export declare function foo(p?: (ip: number | undefined, v: number) => void): void;
+export declare function foo(p?: (ip: number, v: number) => void): void;
 export declare class Bar2 {
     readonly r = 1;
     f: number;
 }
